zoo-css-challenge/style.css

```
@@ -105,25 +110,44 @@ footer {
 
 .image-container img {
   vertical-align: top;
-  border: 5px solid #747c92;
+  border: 5px solid #3660d3;
 }
 
+.image-container img:nth-child(2) {
+  vertical-align: top;
+  border: 5px solid #dd2626;
+}
 .image-container img:first-of-type {
-  vertical-align: bottom;
+  vertical-align: top;
+  border: 5px solid #abdf88;
+
+
 }
 
 .tigers {
   background-color: #0b3c49;
   color: #f0e100;
 }
+.tigers h2, h3{
+  color:#F9CDCD !important;
+}
 
 .giraffe {
-  background-color: #465775;
-  color: #56e39f;
+  background-color: #483C46;
+  color: #BEEE62;
+}
+.giraffe p:nth-child(1){
+font-size:larger ;
+
 }
 
 .giraffe h2 {
-  color: #ef6f6c;
+  color: #F4743B;
+}
+.news h2{
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

```suggestion
  text-align: center;
```
Alternatively, we could use `text-align: center;` here